keps/sig-apps/20190318-Graduate-CronJob-to-Stable.md

```diff
@@ -0,0 +1,98 @@
+---
+title: graduate-cronjob-to-stable
+authors:
+  - "@barney-s"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-scheduling
+reviewers:
+  - "@liggitt"
+  - "@kow3ns"
+  - "@janetkuo"
+  - “@mortent”
+approvers:
+  - "@kow3ns"
+  - "@liggitt"
+editor: TBD
+creation-date: 2019-04-18
+last-updated: 2019-04-18
+status: implementable
```

Comment:
It might be worth making this `provisional` and filling in the answers to the questions by @liggitt. That way we can merge more often.

Reply:
made it provisional